Utilities/StorageFactory/plugins/BuildFile.xml

```diff
@@ -3,3 +3,14 @@
   <flags EDM_PLUGIN="1"/>
   <flags cppflags="-D_FILE_OFFSET_BITS=64"/>
 </library>
+
+<library file="StorageTracerProxy.cc StorageAddLatencyProxy.cc" name="UtilitiesStorageFactoryProxyPlugins">
+  <use name="FWCore/Concurrency"/>
+  <use name="FWCore/MessageLogger"/>
+  <use name="FWCore/ParameterSet"/>
+  <use name="Utilities/StorageFactory"/>
+  <use name="boost"/>
+  <use name="fmt"/>
+  <flags EDM_PLUGIN="1"/>
+  <flags cppflags="-D_FILE_OFFSET_BITS=64"/>
```

Comment:
maybe a comment saying why this flag?

Reply:
I don't really know why add this flag in this way here and there (and not e.g. globally). I included it here because it is used for the other plugins of this package.